### ISSUE TITLE
Upload progress doesn't update past initial state

### ISSUE THREAD

**Issue by stellanor** [RxSwiftCommunity/RxAlamofire#88]
Hi, I haven't been able to get upload progress (based on this sample code) to work:

```
// just fire upload and display progress
_ = upload(Data(), urlRequest: try! RxAlamofire.urlRequest(.get, stringURL))
    .flatMap {
        $0
        .validate(statusCode: 200 ..< 300)
            .validate(contentType: ["text/json"])
            .rx.progress()
    }
    .observeOn(MainScheduler.instance)
    .subscribe { print($0) }
```

My `onNext` only fires once with the initial progress:
```
 RxProgress
  - bytesWritten : 0
  - totalBytes : 0
```

I notice that there was a `rx.progress()` extension added to `UploadRequest` in 09e02d99945fd7c46933729ea5df5e8402e575f2 but it was removed shortly thereafter in fa98bd027fb3c9dc9312d92e6025fbd6c8e3f8e7 . When I revert fa98bd027fb3c9dc9312d92e6025fbd6c8e3f8e7, the upload progress works as expected. Was this a mistake, or is there something I'm missing?

**Comment by tinblanc** [comment]
me too, help plz!

**Comment by freak4pc** [comment]
@stellanor @tinblanc Sorry for taking so long to respond. Would you mind trying the fix from here and let me know if it helps? 

https://github.com/RxSwiftCommunity/RxAlamofire/pull/101

**Comment by freak4pc** [comment]
Fixed by #101
